rust/xain-grpc-api/src/bin/client.rs

```diff
@@ -0,0 +1,62 @@
+use log::info;
+
+use xain_grpc_api::logging;
+
+use std::sync::Arc;
+
+use grpcio::{ChannelBuilder, ChannelCredentials, ChannelCredentialsBuilder, EnvBuilder};
+use xain_proto::coordinator::RendezvousRequest;
+use xain_proto::coordinator_grpc::CoordinatorClient;
+
+use clap::{App, Arg};
+
+type AppError = Box<dyn std::error::Error + Send + Sync>;
+
+fn load_certificates(args: &clap::ArgMatches) -> Result<ChannelCredentials, AppError> {
+    // TODO: load certificates dynamically
```

Comment:
What does this TODO suggest? How would we load certificates dynamically?

Reply:
Heh, missed removing it. The lines below used hard-coded filenames.